packages/verification-vue-component/src/components/CertifactionVerification.vue

```diff
@@ -1,23 +1,35 @@
 <template>
-    <div class="certifaction-verification">
-        <VerificationDemo v-if="demo !== false" @verify-demo="verifyDemo" @dragging-demo-doc="onDraggingDemoDoc"/>
-
-        <div v-if="filteredVerificationItems.length" class="verification-item-list" ref="results">
-            <VerificationItem
-                v-for="verificationItem in filteredVerificationItems"
-                :key="verificationItem.hash"
-                :verificationItem="verificationItem"/>
-        </div>
+    <div class="certifaction-verification"
+         @dragover.prevent="dragOver"
+         @dragleave="dragLeave"
+         @drop="handleDrop">
 
         <VerificationDropBox
-            @files-dropped="verify"
+            :class="{hidden: !dropbox.draggingOver}"
+            :first-verification="filteredVerificationItems.length === 0"
+            @files-selected="verify"
             @drop="drop"/>
 
-        <div class="powered-by">
-            <span class="label">{{ _$t('verification.poweredBy.label') }}</span>
-            <a href="https://certifaction.io" target="_blank">
-                <img src="../assets/img/certifaction_logo.svg" alt="Certifaction"/>
-            </a>
+        <div :class="{hidden: dropbox.draggingOver}">
+            <VerificationDemo v-if="demo !== false" @verify-demo="verifyDemo" @dragging-demo-doc="onDraggingDemoDoc"/>
+
+            <div v-if="filteredVerificationItems.length" class="verification-item-list" ref="results">
+                <VerificationItem
+                    v-for="verificationItem in filteredVerificationItems"
+                    :key="verificationItem.hash"
+                    :verificationItem="verificationItem"/>
+            </div>
+
+            <VerificationFileSelector @files-selected="verify"
+                                      :first-verification="filteredVerificationItems.length === 0"
+            />
+
+            <div class="powered-by">
+                <span class="label">{{ _$t('verification.poweredBy.label') }}</span>
+                <a href="https://certifaction.io" target="_blank">
```

Comment:
As we want to use only the .com URL in the future, we could directly use .com here from now on. At the moment it will redirect to .io

Reply:
Changed it (also in the README)